src/yang/ietf-bgp-policy.yang

```diff
@@ -248,6 +253,92 @@ module ietf-bgp-policy {
           }
         }
       }
+
+      container prefix-sets {
+        description
+          "Definitions for a list of IPv4 or IPv6 prefixes which
+           are matched as part of a policy.";
+
+        list prefix-set {
+          key "name";
+          description
+            "List of the defined prefix sets";
+
+          leaf name {
+            type string;
+            description
+              "Name of the prefix set used to reference the set in
+               match conditions.";
+          }
+
+          leaf mode {
+            type enumeration {
+              enum ipv4 {
+                description
+                  "Prefix set contains IPv4 prefixes only.";
+              }
+              enum ipv6 {
+                description
+                  "Prefix set contains IPv6 prefixes only.";
+              }
+              enum mixed {
+                description
+                  "Prefix set contains mixed IPv4 and IPv6 prefixes";
+              }
+            }
+            description
+              "Indicates the mode of the prefix set, in terms of
+               which address families (IPv4, IPv6, or both) are
+               present. The mode provides a hint, but the device
+               must validate that all prefixes are of the indicated
+               type, and is expected to reject the configuration if
+               there is a discrepancy.  The 'mexed' mode may not be
+               supported on devices that require prefix sets to be
+               of only one address family.";
+          }
+        }
+
+        container prefixes {
+          description
+            "Enclosing container for the list of prefixes in a policy
+             prefix list.";
+
+          list prefix {
+            key "ip-prefix masklength-range";
+            description
+              "List of prefixes in the prefix set.";
+
+            leaf ip-prefix {
+              type inet:ip-prefix;
+              mandatory true;
+              description
+                "The prefix member in CIDR notation. While the
+                 prefix may be either IPv4 or IPv6, most
+                 implementations require all members of the prefix
+                 set to be the same address family. Mixing address
+                 types in the same prefix set is likely to cause an
+                 error.";
+            }
+
+            leaf masklength-range {
+              type string {
+                pattern '^(([0-9]+\.\.[0-9]+)|exact)$';
```

Comment:
What do people feel about "longer" and "or-longer" as additional easy forms?

Reply:
Also, regexes can't count.  General advice is that the first part of the range must be less than or equal to the second part of the range.  How do we want to note that?

Reply:
That is correct. my suspicion is that the range check would be hard. *If* we want, we can move it in the validation part.

Reply:
To make this a concrete request:
add named options longer and or-longer
add to the description that number first number must be <= second number and that first number must be >= the prefix-length.

Reply:
> What do people feel about "longer" and "or-longer" as additional easy forms?

I added "longer" and "or-longer" as additional options in the mask length options, but examples on the web seem to indicate use of "le" or "ge". What is preferable?